.eslintrc

```
@@ -1,13 +1,24 @@
 {
-  "plugins": ["prettier"],
-  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "prettier"
+  ],
```

I believe we have to add `@typescript-eslint` plugin here ([reference](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md#configuration))
```suggestion
  "plugins": [
    "@typescript-eslint",
    "prettier"
  ],
```